File: 2st/Riyeon/문자열 계산하기.py
def solution(my_string):
    li = my_string.split(' ')
    answer = int(li[0])
    for i in range(1, len(li), 2):
        if li[i] == '+':
            answer += int(li[i+1])
        else:
            answer -= int(li[i+1])
    return answer

File: 4st/Eunseong/문자열 계산하기.java
import java.util.*;

class Solution {
    public int solution(String my_string) {
        int answer = 0;
        boolean isPlus = true;
        String pol[] = my_string.split(" ");
        for (int i = 0; i < pol.length; i++) {
            if (i % 2 == 1) {
                if (pol[i].equals("+")) {
                    isPlus = true;
                } else {
                    isPlus = false;
                }
            } else {
                if (isPlus) {
                    answer += Integer.valueOf(pol[i]);
                } else {
                    answer -= Integer.valueOf(pol[i]);
                }
            }
        }
        return answer;
    }
}